Purchases/Networking/Operations/Handling/SubscriberAttributesMarshaller.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriberAttributesMarshaller.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class SubscriberAttributesMarshaller {
+
+    func subscriberAttributesToDict(subscriberAttributes: SubscriberAttributeDict) -> [String: [String: Any]] {
```

Comment:
This might be a bit nit picky but... something about this method name with the class named and name parameters feels about smurfy 🤔 The types are described in the named parameter and the return type. Can this be renamed to something like `mapToDict` or `map`?

Reply:
I ♥️ `map`, updated!